src/preferences/PreferencesManager.js

```diff
@@ -123,7 +125,7 @@ define(function (require, exports, module) {
         // having _doNotCreate set to true, then the caller is using the old preferences model.
         if (!_doNotCreate) {
             var clientString = typeof clientID === "object" ? clientID.uri : clientID;
-            DeprecationWarning.deprecationWarning("getPreferenceStorage is called with client ID '" + clientString + ",' use PreferencesManager.definePreference instead.");
+            DeprecationWarning.deprecationWarning("PreferencesManager.getPreferenceStorage() is called with client ID '" + clientString + ",' use PreferencesManager.definePreference() instead.");
```

Comment:
The closing single quote should come before comma after printing the client string.
Also more importantly, we could have a uniform deprecation message instead of three different on various occasions. I would suggest a simple and uniform:-
"PreferencesManager.\<some_deprecated_fn\> called with client id \<client_id\> has been deprecated. Use PreferencesManager.\<updated_fn\> instead".